pulp_ci_centos/Containerfile

```diff
@@ -21,27 +21,25 @@ ENV PULP_SETTINGS=/etc/pulp/settings.py
 ENV _BUILDAH_STARTED_IN_USERNS=""
 ENV BUILDAH_ISOLATION=chroot
 
-# use python38
-RUN dnf -y module disable python36
-RUN dnf -y module enable python38
-
 # glibc-langpack-en is needed to provide the en_US.UTF-8 locale, which Pulp
 # seems to need.
 #
 # The last 5 lines (before clean) are needed until python3-createrepo_c gets an
 # RPM upgrade to 0.16.2. Until then, we install & build it from PyPI.
 RUN dnf -y install dnf-plugins-core && \
-    dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
-    dnf config-manager --set-enabled powertools && \
-    dnf -y module enable postgresql:12 && \
+    dnf erase subscription-manager -y  && \
+    dnf config-manager --set-enabled crb && \
+    dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
     dnf -y update && \
-    dnf -y install python38 python38-devel && \
+    dnf -y install python-devel && \
     dnf -y install openssl && \
     dnf -y install wget git && \
     dnf -y install python3-psycopg2 && \
     dnf -y install glibc-langpack-en && \
     dnf -y install python3-libmodulemd && \
     dnf -y install python3-libcomps && \
+    dnf -y install https://kojihub.stream.centos.org/kojifiles/packages/tzdata/2021e/1.el9/noarch/tzdata-2021e-1.el9.noarch.rpm && \
+    dnf -y install tzdata && \
```

Comment:
Hack to fix tzdata issue:
https://gitlab.com/redhat/centos-stream/release-engineering/kickstarts/-/blob/main/CentOS-Stream-9-container-base.ks#L47-50
 "if you want to change the timezone, bind-mount it from the host or reinstall tzdata"